sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/statsEstimation/StatsEstimationSuite.scala

```diff
@@ -48,6 +77,14 @@ class StatsConfSuite extends StatsEstimationTestBase {
     // Return the simple statistics
     assert(plan.stats(conf.copy(cboEnabled = false)) == expectedDefaultStats)
   }
+
+  /** Check estimated stats which is the same when cbo is turned on/off. */
+  private def checkStats(plan: LogicalPlan, expected: Statistics): Unit = {
```

Comment:
You know, this is a utility function. We can make it more general by having two expected stats values